shared/macros-oval.jinja

```diff
@@ -218,6 +218,32 @@
   </unix:file_test>
 {{%- endmacro %}}
 
+{{#
+  Macro to define the OVAL criterion that requires a file not to exist.
+  This macro can take one parameter:
+    - filepath (String): Path to the file to be checked.
+    - stem of the test name
+#}}
+{{%- macro oval_config_file_absent_criterion(filepath, stem) -%}}
+	<criterion comment="Pass if {{{ filepath }}} does not exist" test_ref="test_{{{ stem }}}" />
+{{%- endmacro %}}
+
+{{#
+  Macro to define the OVAL test to check if the configuration file exists (Test definition).
+  This macro can take one parameter:
```

Comment:
```suggestion
  This macro can take up to four parameters:
```

Reply:
I went for a copy-paste-friendly "Parameters:"

Reply:
OK, thanks, great idea!